2017-05-17 10:20:53.606 -0600	START		Running practice_script.script
2017-05-17 10:20:54.144 -0600	click	terminal_icon	at (155, 1037)	1	on_practice_script	practice_script.script
2017-05-17 10:20:54.867 -0600	typetext		ls	2	on_practice_script	practice_script.script
2017-05-17 10:20:55.580 -0600	typetext		[return]	3	on_practice_script	practice_script.script
2017-05-17 10:20:56.734 -0600	typetext		 cd Dropbox/github/connect_four/app/bin 	4	on_practice_script	practice_script.script
2017-05-17 10:20:57.448 -0600	typetext		[return]	5	on_practice_script	practice_script.script
2017-05-17 10:20:58.251 -0600	typetext		./main.py	6	on_practice_script	practice_script.script
2017-05-17 10:20:58.353 -0600	EndTestCase		(Duration:"4.742096", Errors:"1", Exceptions:"0", StartTime:"2017-05-17 10:20:53 -0600", Successes:"0", TestCase:"practice_script.script", Warnings:"0")
2017-05-17 10:21:00.755 -0600	FAILURE	Screen_Error.png	STUnknownMessage ERROR: No Such Function: 'returnimagefound' -- Execution Time 0:00:05 practice_script.script
